### DISCUSSION TITLE
Remove a dependency with transient dependencies

### DISCUSSION THREAD

**Discussion by mdbergmann** [ocicl/ocicl#26] [General]
Hello, I'm trying ocicl.

It seems like ocicl downloads add transient dependencies into 'systems' folder and populates also 'systems.csv' with them.
What is the procedure when a main dependency is removed? It should be easy to remove it manually from 'systems.csv', but what about the transient dependencies?
How to remove the ones that are not used anymore?

**Comment by atgreen** [comment]
If you really just want to clean it up, I would simply delete systems
and systems.csv, and then load my program again with
ocicl-runtime:*download* set to t.

**Reply by mdbergmann** [reply]
k, thanks.
